trunk-recorder/main.cc

```diff
@@ -659,9 +663,9 @@ bool start_recorder(Call *call, TrunkMessage message, System *sys) {
       //int total_recorders = get_total_recorders();
 
       if (recorder) {
-        if (message.meta.length()) {
-          BOOST_LOG_TRIVIAL(trace) << message.meta;
-        }
+        /* if (message.meta.length()) {
```

Comment:
I am confused why this section was commented out. If it is too verbose for normal use, lets leave it as TRACE vs DEBUG

Reply:
As shown in the commit https://github.com/robotastic/trunk-recorder/commit/917abd9f3fc427c966cb9b79670923df4a2c395b I'm sending messages to the log in p25_parser.cc, as in before returning to main.cc and this line. This allows different messages to be logged at different severity levels.

Reply:
Gotcha - Can you delete the if statement instead of commenting it?

Reply:
Will do.